pygeoapi/process/base.py

```diff
@@ -49,14 +51,19 @@ def __init__(self, processor_def: dict, process_metadata: dict):
         """
         self.name = processor_def['name']
         self.metadata = process_metadata
+        self.supports_outputs = False
 
-    def execute(self, data: dict) -> Tuple[str, Any]:
+    def execute(self, data: dict, outputs: Optional[dict] = None
+                ) -> Tuple[str, Any]:
```

Comment:
@ricardogsilva
As stated during the last Sprint in Evora, pygeoapi is _currently not intended to handle multiple outputs_, unless it will become mandatory (i.e., if I understand correctly, only if CITE will test for it).
Additionally, trying to minimize the impact on currently working development is a must.

In case multiple outputs will be handled, in my opinion there is not a single solution on how to implement it.
Having the process returning a list of outputs and their media type is one solution.
Another solution could be **the process continues returning a single object** with several properties, each property being one of the requested outputs; viable solutions exists to pass back to the manager the media-type of each output.

The aim of this PR is to make a first step to solve issue #1420 only: _propagating parameter outputs_.
The actual reason for this is to allow proper handling of the property **transmissionMode**, which can be done within the manager.
If this PR will be accepted and merged, then I will rise an issue and try to contribute to implement a strategy to handle the property **transmissionMode** within the manager.

Adding **support for handling multiple output** is welcome, but it **is beyond the scope of this PR** and the original issue.

Reply:
@ricardogsilva 
I changed and committed as suggested, but I got the following error:
```
 flake8_py3: pygeoapi/process/base.py#L57
[E125] continuation line with same indent as next logical line
```

Please advise.